modules/gtdbtk/classifywf/main.nf

```diff
@@ -0,0 +1,60 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options    = initOptions(params.options)
+
+process GTDBTK_CLASSIFY {
+    tag "${meta.assembler}-${meta.id}"
+
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['assembler', 'id']) }
```

Comment:
As of now I have re-used the code available at https://github.com/nf-core/mag/blob/master/modules/local/gtdbtk_classify.nf.